boards/pico/src/lib.rs

```diff
@@ -33,7 +33,7 @@ hal::bsp_pins!(
     Gpio20 { name: gpio20 },
     Gpio21 { name: gpio21 },
     Gpio22 { name: gpio22 },
-    Gpio23 { name: bootsel },
+    Gpio23 { name: b_power_save },
```

Comment:
Just curious: What does the prefix `b_` mean?
Or is it a typo and should be `n_power_save` because of negative logic, low level means power save is active?

Reply:
b for bar. Same as n.